pkg/virt-controller/services/template.go

```diff
@@ -1760,6 +1760,16 @@ func addProbeOverhead(probe *v1.Probe, to *resource.Quantity) bool {
 }
 
 func updateProbe(vmi *v1.VirtualMachineInstance, computeProbe *k8sv1.Probe) {
+	if vmi.Spec.ReadinessProbe == nil && vmi.Spec.LivenessProbe == nil {
```

Comment:
this check should not be necessary as we already check that when we call `updateProbe` (we pass it a probe that should not be nil).

Reply:
Removed now